### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by Diogo-Rossi**
Update docs do reflect [the optional requirements](https://abqpy.com/en/latest/getting_started.html#optional-requirements)

### CODE REVIEW

**Comment by haiiliin** [comment]
Do you think we need to simplify the optional requirements in `README.md`?

File: docs/source/getting_started.rst [hunk 1]
```diff
@@ -156,10 +156,17 @@ Install A Specific Version
...
+
+    pip install abqpy==2022.3.1
+
+Using `conda`:
+
 .. code-block:: sh
 
-    pip install abqpy==2022.3.2
-    conda install -c haiiliin abqpy=2022.3.2
+    conda install -c haiiliin abqpy=2022.3.1
```
Comment (haiiliin) [inline docs/source/getting_started.rst:169]: The latest is now 2022.3.2

Comment (Diogo-Rossi) [reply docs/source/getting_started.rst:169]: sorry, my bad

**Comment by Diogo-Rossi** [comment]
> Do you think we need to simplify the optional requirements in `README.md`?

You mean, like this [37095e5](https://github.com/haiiliin/abqpy/pull/1434/commits/37095e5934956c3ef8e90445cd9823035ae8113c)?

**Comment by haiiliin** [comment]
Yes, make it simple

**Comment by Diogo-Rossi** [comment]
> Yes, make it simple

Yes, I think so.

**Comment by Diogo-Rossi** [comment]
> Do you think we need to simplify the optional requirements in `README.md`?

Done in #1476

**Pull request merged**